src/modules/network/task.hpp

```diff
@@ -27,6 +27,14 @@ struct target_t
     std::optional<std::string> interface;
 };
 
+struct synchronizer_t
+{
```

Comment:
I don't see ratio_reads/ratio_writes get modified.
Why is there a need to make them atomic?

Reply:
ratio_reads/ratio_writes are changing each worker iteration, and can be read at other thread. So atomic is required here

Reply:
It is full copy of such option in block module